x-pack/legacy/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -28,6 +28,7 @@ export const REACT_APP_ROOT_ID = 'react-apm-root';
 const MainContainer = styled.main`
   min-width: ${px(unit * 50)};
   padding: ${px(units.plus)};
+  height: 100%;
```

Comment:
This was needed to vertically centralize the message `Fetching trace...` in the transition page.